packages/testing-helpers/src/scopedElementsWrapper.js

```diff
@@ -58,10 +75,12 @@ customElements.define('scoped-elements-test-wrapper', ScopedElementsTestWrapper)
  * @returns {TemplateResult}
  */
 export function getScopedElementsTemplate(template, scopedElements) {
-  return html`
-    <scoped-elements-test-wrapper
-      .scopedElements="${scopedElements}"
-      .template="${template}"
-    ></scoped-elements-test-wrapper>
-  `;
+  const strings = [
+    `<${wrapperTagName} .scopedElements="`,
+    '" .template="',
+    `"></${wrapperTagName}>`,
+  ];
+
+  // @ts-ignore
```

Comment:
`@ts-ignore` is required because `html` expects a `TemplateStringsArray` instead of a `string[]`